src/command_modules/azure-cli-container/azure/cli/command_modules/container/_params.py

```diff
@@ -29,13 +29,25 @@ def _environment_variables_type(value):
     return {'name': env_name, 'value': env_value}
 
 
+def _secure_environment_variables_type(value):
```

Comment:
Use this instead? https://github.com/Azure/azure-cli/blob/dev/src/azure-cli-core/azure/cli/core/commands/validators.py#L55

Reply:
That would require having a function inside the create command to convert from that dictionary to a dictionary with the form {'name': "name", 'value': "value"} for env vars and {'name':"name", "secureValue": "value"} for secure values. I like having that logic in params parsing but open to whatever you would prefer.

Reply:
Why don't you use the validator to achieve this? Just like how we process tags. We'd like the user experience of specifying key-value pairs consistent across commands.

Reply:
if we change it to use the above validator it would be a breaking change to our customers. That uses a ";" to seperate where we have been using a spaces.